ios/RNGestureHandlerManager.m

```diff
@@ -159,7 +159,9 @@ - (void)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
 
     UIView *parent = rootContentView.superview;
     if ([parent isKindOfClass:[RCTRootView class]]) {
-        [(RCTRootView*)parent cancelTouches];
+      // Previously we were using cancelTouches method from RCTRootView.
+      // Now it's deprecated and we call cancel directly on touchHandler of contentView
+      [(RCTTouchHandler *)[[parent valueForKey:@"contentView"] valueForKey:@"touchHandler"] cancel];
```

Comment:
I don't like that we use `valueForKey` here. IMO it is more likely that this code will break comparing to the use of deprecated method.

If we want to access `_contentView` can we maybe instead of looking for `RCTRootView` parent look for `RCTRootContentView`? I understand that _contentView is a direct subview of the rootview and that should work